src/components/atoms/ClientPortal/ClientPortal.tsx

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useRef } from "react";
+import { createPortal } from "react-dom";
+type ClientPortalInterface = {
+  children: React.ReactNode;
+  show?: boolean;
+  onClose?: () => void;
+};
+const ClientPortal = ({ children, show }: ClientPortalInterface) => {
+  const ref = useRef<Element | null>(null);
+
+  useEffect(() => {
+    ref.current = document.body;
```

Comment:
We can use this for now , but ultimately I still think it would be better to explicity pass the dom element for the portal as a prop